lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -608,16 +610,6 @@ def follow_commentable(request, course_id, commentable_id):
 @require_POST
 @login_required
 @permitted
-def follow_user(request, course_id, followed_user_id):
```

Comment:
Have you confirmed with Leslie that we won't want this functionality in the future? I get wanting to delete it if we're not using it right now, but it looks like it's just a thin view wrapper around already existing functionality here.